Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -157,6 +161,10 @@ class BrowserViewController: UIViewController {
                previousTraitCollection.horizontalSizeClass != .Regular
     }
 
+    func shouldShowTopTabsForTraitCollection(previousTraitCollection: UITraitCollection) -> Bool {
```

Comment:
I think the name of the parameter is wrong and should be `newTraitCollection` since that's what were passing in in the `updateToolbarStateForTraitCollection` method.